OneCommunityGlobal/HighestGoodNetworkApp
# Changhao - added more unit tests for activeinactiveconfirmationpopup.jsx

# Description
more unit tests for activeinactiveconfirmationpopup.jsx

## How to test:
1. check into current branch
2. do `npm install` and `npm test ActiveInactiveConfirmationPopup.test.js` to run this PR locally
3. verify all 12 tests passed

## Screenshots:
<img width="682" alt="Screenshot 2024-02-22 at 11 43 07 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/104778270/d0bf6dae-b3c2-48f4-8539-ea95de63ad28">
